src/tile.cpp

```diff
@@ -565,17 +565,17 @@ ReturnValue Tile::queryAdd(int32_t, const Thing& thing, uint32_t, uint32_t flags
 		}
 
 		const CreatureVector* creatures = getCreatures();
-		if (const Player* player = creature->getPlayer()) {
+		if (auto player = creature->getPlayer()) {
```

Comment:
Could this be `const auto`
```suggestion
		if (const auto player = creature->getPlayer()) {
```